### DISCUSSION TITLE
Handwritten name on my school id

### DISCUSSION THREAD

**Discussion by pimglar** [community/community#49824] [GitHub Education]
I have an issue with my application, because every time I apply it's auto-rejecting because:
![obraz](https://user-images.githubusercontent.com/119127477/224508152-f150ba6f-1d3c-4367-a5f9-63ac8deaa1b5.png)

I think it is because on my school ID, my name is handwritten and may not be readable. How can I prove to GitHub that the name on the school ID is actually my name?

**Comment by Ntshangase** [comment]
Hello, Maybe you can try providing an **enrollment verification letter**,

For more information on other documents you can apply with, Check out: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

**Comment by bbkx226** [comment]
If you believe that the auto-rejection of your application is due to the name on your school ID being handwritten and potentially unreadable, there are a few things you can try to prove to GitHub that the name on your ID is in fact your name:

1. Provide additional documentation: In addition to your school ID, you can provide additional documentation that confirms your name, such as a driver's license, passport, or birth certificate. You can upload these documents to GitHub and provide them as evidence that your name on your school ID is correct.

2. Ask your school for a new ID: If possible, you can ask your school to provide you with a new ID that has your name printed instead of handwritten. This may make it easier for GitHub to verify your identity.

Ultimately, the goal is to provide enough evidence to GitHub to confirm that the name on your school ID is your name. It may take some time and effort, but with persistence, you should be able to resolve this issue.
